src/main/java/org/jenkinsci/plugin/gitea/GiteaSCMFileSystem.java

```diff
@@ -152,7 +152,7 @@ public SCMFileSystem build(@NonNull SCMSource source, @NonNull SCMHead head, @Ch
                     CredentialsProvider.lookupCredentials(
                             StandardCredentials.class,
                             owner,
-                            Jenkins.getAuthentication(),
+                            ACL.SYSTEM,
```

Comment:
In other plugins I noticed that they check if owner instance of Queue.Task:
```suggestion
                            owner instanceof Queue.Task
                                    ? ((Queue.Task) context).getDefaultAuthentication()
                                    : ACL.SYSTEM,
```

Don't know the use case for this tho :/

Reply:
I'm not quiet sure what this is for either, but added it anyway...